src/misc/lv_anim_timeline.c

```diff
@@ -186,6 +187,20 @@ static void anim_timeline_set_act_time(lv_anim_timeline_t * at, uint32_t act_tim
             if(a->exec_cb) a->exec_cb(a->var, value);
             if(a->custom_exec_cb) a->custom_exec_cb(a, value);
         }
+ 
+        if(at->reverse) {
+            if(value > a->start_value) at->anim_dsc[i].flag_completed = false;
+            else if (value == a->start_value) {
```

Comment:
We should use the `time` instead of the value as it might happen that the current value is equal to the end value earlier too. E.g. in case of an over shooting animation.